docs/source/tutorials/tabular.ipynb

```diff
@@ -391,7 +391,7 @@
    "source": [
     "# Hidden code cell to check that cleanlab has improved prediction accuracy\n",
     "if acc_og >= acc_cl:\n",
-    "    raise Exception(\"Cleanlab training failed to improve model accuracy.\")"
+    "    raise Exception(\"Cleanlab training failed to improve model accuracy.\")\n"
```

Comment:
this \n is causing notebook linter to fail.

Reply:
Why is there a change to this notebook at all?

Reply:
fixed